commands/banintro.js

```diff
@@ -0,0 +1,35 @@
+//Call: Slash command banintro
+//Removes a selected users intro
+const fs = require('fs');
+
+exports.commandName = 'banintro';
+
+exports.run = async (client, interaction) => {
+    //TODO add permissions check?
+
+    if(!interaction.member.permissionsIn(interaction.channel).has("ADMINISTRATOR")) {
+        interaction.reply({content:"This command is only for people with Administrator permissions", ephemeral:true}); 
+        return;
+    }
+    let badUser = interaction.options.getUser('user');
+    if(fs.existsSync(`./data/intros/${interaction.guild.id}/${badUser.id}.mp4`)){
+       fs.unlinkSync(`./data/intros/${interaction.guild.id}/${badUser.id}.mp4`);
```

Comment:
We should not be using synchronous functions, like I said on discord. I would recommend using the asynchronous version of fs.unlink, then just do nothing if it returns an error, instead of checking if the file exists beforehand.

Reply:
should I make the whole function synchronous then

Reply:
Is this a joke? If it isn't, no. Use fs.unlink (the one that's asynchronous, hence no Sync at the end) and handle the error in the callback. You can read up on fs.unlink on the node.js docs